CMakeLists.txt

```diff
@@ -22,4 +28,10 @@ target_link_libraries(QuickDashBoard
         OpenSSL::Crypto
         OpenSSL::SSL
         Python::Python
+)
+
+set_target_properties(QuickDashBoard PROPERTIES

```

Comment:
This should be set for each target and ensures executables are placed in "${CMAKE_BINARY_DIR}/bin" and shared object/dynamic libraries and static libraries are placed in "${CMAKE_BINARY_DIR}/lib"